src/components/results/results.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import DoctorInfoList from './doctorInfoList.js';
+import TotalResults from './totalResults.js';
+
+
+
+class Results extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {      
```

Comment:
[question] What was your reasoning for keeping application state in the results component? Another way to think about this: if your main application state is in a child "results" component, how do you plan to share it as props to parent and sibling components?